src/simtools/db/db_handler.py

```diff
@@ -541,63 +510,24 @@ def get_simulation_configuration_parameters(
             if simulation_software is not valid.
         """
         if simulation_software == "corsika":
-            return self.get_corsika_configuration_parameters(model_version)
+            return self.get_model_parameters(
+                None,
+                None,
+                model_version,
+                collection="configuration_corsika",
+            )
```

Comment:
This doesn't depend on the site (or more accurately, the atmospheric profile dependence is in a different table), so setting the site/array_element to `None` makes sense. However, it reminded me that some of the parameters here depend on the primary (muons). How do we deal with it? We can postpone this issue for later of course, but if you agree that we should deal with it, I will open an issue so we don't forget.

Reply:
Please open an issue.

Reply:
Done (#1328).